CMakeLists.txt

```diff
@@ -200,6 +200,9 @@ if(BUILD_GSTREAMER_PLUGIN)
 
   add_executable(kinesis_video_gstreamer_audio_video_sample_app samples/kinesis_video_gstreamer_audio_video_sample_app.cpp)
   target_link_libraries(kinesis_video_gstreamer_audio_video_sample_app ${GST_APP_LIBRARIES} KinesisVideoProducer)
+
```

Comment:
Let's try to move this directly to the samples.

Reply:
Do you mean the samples repo, https://github.com/aws-samples/amazon-kinesis-video-streams-demos/tree/samples-folder? 

If yes, I'm also working on it rn. I'm planning to merge this PR, then later transfer it to the demo repo.